assets/src/apps/authoring/components/PropertyEditor/custom/CompJsonEditor.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { CSSProperties, useState } from 'react';
+interface JsonEditoProps {
+  jsonValue: any;
+  onChangeHandler: any;
+}
+const CompJsonEditor: React.FC<JsonEditoProps> = (props: any) => {
+  const { jsonValue, onChangeHandler } = props;
+  const val = { id: jsonValue.id, custom: jsonValue.custom };
+  const [value, setValue] = useState<string>(JSON.stringify(val, null, 4));
+  const popup: CSSProperties = {
```

Comment:
perhaps we should use a bootstrap modal instead since it already has the backdrop support and whatnot